src/components/common/Extensions/UserNavigation/partials/UserMenu/UserMenu.tsx

```diff
@@ -8,28 +8,108 @@ import Link from '~shared/Link';
 import Icon from '~shared/Icon';
 import ThemeSwitcher from '~common/Extensions/ThemeSwitcher';
 import styles from './UserMenu.module.css';
+import PopoverBase from '~shared/Extensions/PopoverBase';
+import TitledContent from '~common/Extensions/TitledContent/TitledContent';
+import Avatar from '~shared/Extensions/Avatar';
+import { useCopyToClipboard } from '~hooks/useCopyToClipboard';
+import Token from '../Token';
+import UserAvatar from '~shared/Extensions/UserAvatar';
+import MemberReputation from '../MemberReputation';
 
 const displayName = 'common.Extensions.UserNavigation.partials.UserMenu';
 
-const UserMenu: FC<UserMenuProps> = ({ tooltipProps, setTooltipRef, isWalletConnected }) => {
+const UserMenu: FC<UserMenuProps> = ({
+  tooltipProps,
+  setTooltipRef,
+  isWalletConnected,
+  userName,
+  isVerified,
+  walletAddress,
+  copyUrl,
+  userReputation,
+  totalReputation,
+  nativeToken,
+}) => {
   const isMobile = useMobile();
   const { formatMessage } = useIntl();
   const { connectWallet } = useAppContext();
+  const { handleClipboardCopy } = useCopyToClipboard(walletAddress || '');
 
   return (
-    <div
-      ref={setTooltipRef}
-      {...tooltipProps({
-        className: clsx('px-0 md:px-6 py-6 flex justify-start z-[9999] tooltip-container', {
-          'w-full border-none shadow-none': isMobile,
-          'w-[20.125rem]': !isMobile,
-          'h-[32rem] md:h-[23rem]': !isWalletConnected,
-          'md:h-[28.125rem]': isWalletConnected,
-        }),
+    <PopoverBase
+      setTooltipRef={setTooltipRef}
+      tooltipProps={tooltipProps}
+      classNames={clsx('px-0 md:px-6 py-6 flex justify-start z-[9999] tooltip-container', {
+        'w-full border-none shadow-none': isMobile,
+        'w-[20.125rem]': !isMobile,
+        'h-[32rem] md:h-[23rem]': !isWalletConnected,
+        'md:h-[31.5rem]': isWalletConnected,
       })}
     >
       {isWalletConnected ? (
-        <>Connected component</>
+        <div>
+          <div className={styles.mobileButtons}>
+            {nativeToken && <Token nativeToken={nativeToken} />}
+            <Button mode="tertiaryOutline" isFullRounded>
+              <div className="flex items-center gap-3">
+                <UserAvatar userName="panda" size="xxs" isLink={false} />
```

Comment:
`<UserAvatar userName="panda" size="xxs" isLink={false} />`
`isLink={false}`  I think prop `isLink` should be false as default. What do you  think?
I know that it's not your code, but  can you look on that?